templates/configmaps_st2-conf.yaml

```diff
@@ -24,6 +24,10 @@ data:
     [coordination]
     url = etcd://{{ index .Values "etcd" "fullnameOverride" }}:2379
     {{- end }}
+    {{- if index .Values "redis" "enabled" }}
+    [coordination]
+    url = redis://{{ .Release.Name }}-redis-node-0.{{ .Release.Name }}-redis-headless:26379?sentinel=mymaster&sentinel_fallback={{ .Release.Name }}-redis-node-1.{{ .Release.Name }}-redis-headless:26379&sentinel_fallback={{ .Release.Name }}-redis-node-2.{{ .Release.Name }}-redis-headless:26379
```

Comment:
Updated the templating here with the proper Release.Name to make sure the connection string is tested in our CircleCI e2e minikube environment.